### ISSUE TITLE
Using repositories from Github Legacy Search as scope for Github Code Analyzer

### ISSUE THREAD

**Issue by raunaqarora** [Netflix/Scumblr#235]
I need to use repositories discovered by Github Legacy Search as input for Github Code Analyzer. I am unable to find any documentation within the Wiki about how I could do this.

**Comment by sbehrens** [comment]
Hi @rahulsom ,

Github Code Analyzer runs on Github projects specified by an org or user name.  Currently there isn't a way to send that specific analyzer an existing Result Filter as input for analysis.

If you could explain a little more about your use case I can see if we have other task types that would help.  

-Scott

**Comment by raunaqarora** [comment]
Hey @sbehrens ,
I'm looking for certain keyword matches across public Github repositories. My initial thought was to use legacy search but legacy search by default just returns a link to repositories matching the keywords. Next, I tried using Github Legacy search with search type parameter set to 'code', but the task would complete without producing any results (Possibly due to too many matches? I couldn't find a way to limit search results within Legacy Search). I then attempted to use the repository links populated in results by Github legacy search as input for Github Code Analyzer which is what caused me to post this question! 

To summarize: I want results to be populated with code from public repositories that contain certain keywords. I cannot use Github Code Analyzer since I do not know which repositories / organization / users, I want to sync. Rather, I need to do this dynamically as I get new matches.
I can do this function using the Github UI - [Example](https://github.com/search?q=%22%40xyz.com%22&type=Code)

Thanks,
Ron

**Comment by sbehrens** [comment]
Hi Ron,

This sounds like a really cool use case that the tool currently doesn't support.  I currently don't think we'll throw this on our personal roadmap due to other commitments.  I'll put a note that we want help to build this out, and hopefully, someone in the community can dedicate a little time to make this happen.
